setup.py

```diff
@@ -156,7 +156,7 @@ def run(self):
         prefix = os.path.abspath(self.prefix)
         self.set_data_files(prefix)
         self.do_egg_install()
-        self.ooniresources()
+        #self.ooniresources()
```

Comment:
Why did you comment out the invocation of ooniresources? Perhaps we should add an option to the setup.py script if the user needs to disable it for some reason. Would that work for you?